src/index.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
 import './index.css';
 import App from './pages/App';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
-  <React.StrictMode>
+  <Router>
     <App />
-  </React.StrictMode>,
+  </Router>,
   document.getElementById('root'),
 );
```

Comment:
Do you think as a part of your initial PR you could configure the current `App` element as its own Route, create a new page in the pages directory (it can just be a copy/paste of the current `App` with a different name), and have that new page render at another Route endpoint (we could go with `/newApp` to start)? Hopefully we should have something closer to the example found [here](https://reactrouter.com/en/main/getting-started/overview#configuring-routes) on the quickstart page.

Reply:
Ya, no problem! I was unsure how far to take the setup since we hadn't agreed upon a structure, so I went with something simple.